tablo/models.py

```diff
@@ -167,6 +167,10 @@ def extent(self):
         return self._extent
 
     @property
+    def srid(self):
+        return json.loads(self.service.spatial_reference)['wkid']
```

Comment:
Should we cache the parsed JSON in a private variable?

```python
    def srid(self):
        if not hasattr(self, '_srid'):
            self._srid = json.loads(self.service.spatial_reference)
        return self._srid['wkid']
```

If this is called a lot it may prevent a lot of string parsing.